services/director-v2/src/simcore_service_director_v2/modules/dynamic_sidecar/api_client/_thin.py

```diff
@@ -27,12 +27,7 @@ def __init__(self, app: FastAPI):
             app.state.settings.DYNAMIC_SERVICES.DYNAMIC_SIDECAR
         )
 
-        self.client = AsyncClient(
-            timeout=Timeout(
-                settings.DYNAMIC_SIDECAR_API_REQUEST_TIMEOUT,
-                connect=settings.DYNAMIC_SIDECAR_API_CONNECT_TIMEOUT,
-            )
-        )
+        self.client = AsyncClient()
```

Comment:
why create ``self.client``  here and you override it in the base class afterwards using ``super().__init__``??

Reply:
removed, this was a mistake